index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
+        <div id="works">
+          <div class="introInfo">
+            <div class="title"><span>Multi-Post Stories</span></div>
+            <p>
+              <span>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+              </span>
+            </p>
+          </div>
+          <div class="buttonGroup">
+            <div class="button">css</div>
+            <div class="button">html</div>
+            <div class="button">bootstrap</div>
+            <div class="button">Ruby</div>
+          </div>
+          <div class="bigButton">See Projects</div>
+        </div>
+        
+        <section id="seeProjects">
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` the project title ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

Reply:
I will find a way to report you. It looks like you have only one way of doing things

Reply:
Hello @til2to I am not making an obstacle for you, if i make you to feel just like that I am really sorry but I suggested the changes to make your  HTML code more semantic correct, but if you have suggestion I am here to discuss and if you want to report you can report.

Reply:
Sorry about that, after the going through critically, you were right. thanks for the corrections